theseus_gui/src/components/ui/Navigation.vue

```diff
@@ -0,0 +1,155 @@
+<script setup>
+import { SearchIcon, BookIcon, ClientIcon, PlusIcon, SettingsIcon } from 'omorphia'
+import { useInstances } from '@/store/state'
+import { RouterLink, useRoute } from 'vue-router'
+import SideNav from '@/components/SideNav.vue'
+import UserSection from '@/components/ui/UserSection.vue'
+import Instance from '@/components/ui/Instance.vue'
+const instances = useInstances()
+
+instances.fetchInstances()
+const route = useRoute()
+</script>
+
+<template>
+  <SideNav class="navigation">
+    <UserSection />
+    <div class="pages-list">
+      <RouterLink
+        to="/"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <ClientIcon />Home</RouterLink
+      >
+      <RouterLink
+        to="/browse"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <SearchIcon />Browse</RouterLink
+      >
+      <RouterLink
+        to="/library"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      >
+        <BookIcon />Library</RouterLink
+      >
+    </div>
+    <div class="instance-list">
+      <p>Instances</p>
+      <Instance v-for="instance in instances.instances" display="list" :instance="instance" />
+    </div>
+    <RouterLink
+      to="/add-instance"
+      class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+      id="add-instance-btn"
+    >
+      <PlusIcon />
+      Create Instance
+    </RouterLink>
+
+    <div id="settings">
+      <RouterLink
+        to="/settings"
+        class="omorphia__button button-base padding-block-sm padding-inline-lg radius-md standard-button"
+        ><SettingsIcon /> Settings</RouterLink
+      >
+    </div>
+  </SideNav>
+</template>
+
+<style lang="scss" scoped>
+.pages-list {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  justify-content: flex-start;
+  width: 100%;
+
+  a {
+    display: flex;
+    align-items: flex-start;
+    width: 80%;
+    margin: 0.3rem auto;
+    text-align: left;
+    font-size: 0.9rem;
+    font-weight: 400;
+    background: inherit;
+    transition: all ease-in-out 0.1s;
+    color: var(--color-primary);
+
+    &:hover {
+      background-color: var(--color-button-bg);
+      box-shadow: 0px 4px 4px rgba(0, 0, 0, 0.25);
+      text-decoration: none;
+    }
+
+    svg {
+      margin-right: 0.25rem;
+    }
+  }
+}
+
+.instance-list {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 70%;
+  margin: 0.4rem;
+
+  & > p {
+    color: var(--color-base);
+    margin: 1rem 0;
+    font-size: 1.2rem;
+    line-height: 13px;
+    font-weight: 500;
+    text-transform: uppercase;
+  }
+}
+
+#settings {
```

Comment:
avoid using IDs and use css classes instead (take this everywhere, don't want to comment everywhere else)